src/app/Classes/LogViewer.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace Backpack\LogManager\app\Classes;
+use Illuminate\Support\Facades\Storage;
+
+/***
+ * Class LogViewer
+ * @package Backpack\LogManager\app\Classes
+ *
+ * Credits https://github.com/rap2hpoutre/laravel-log-viewer
+ */
+
+class LogViewer
+{
+    /**
+     * @var string file
+     */
+    private static $file;
+
+    private static $levels_classes = [
+        'debug'     => 'info',
+        'info'      => 'info',
+        'notice'    => 'info',
+        'warning'   => 'warning',
+        'error'     => 'danger',
+        'critical'  => 'danger',
+        'alert'     => 'danger',
+        'emergency' => 'danger',
+        'processed' => 'info',
+    ];
+
+    private static $levels_imgs = [
+        'debug'     => 'info',
+        'info'      => 'info',
+        'notice'    => 'info',
+        'warning'   => 'warning',
+        'error'     => 'warning',
+        'critical'  => 'warning',
+        'alert'     => 'warning',
+        'emergency' => 'warning',
+        'processed' => 'info',
+    ];
+
+    /**
+     * Log levels that are used.
+     *
+     * @var array
+     */
+    private static $log_levels = [
```

Comment:
Ok.

Where do these come from? These keys seem to appear above as well.

Reply:
I believe it's the difference between the log levels (the package Laravel uses) and the bootstrap classes. They're mapped (ex: "notice" error is shown as "info" blue).